docs/docs/configuration/index.md

```diff
@@ -245,6 +245,9 @@ motion:
 # NOTE: Can be overridden at the camera level
 record:
   # Optional: Enable recording (default: shown below)
+  # WARNING: If automations will be used to control enabling/disabling record,
```

Comment:
If recording is disabled in the config, turning it on via the UI or MQTT later will have no effect.